app/views/challenge/theirs.scala

```diff
@@ -59,14 +59,14 @@ object theirs {
                 frag(
                   hr,
                   badTag(
-                    p("This game is rated"),
+                    p(trans.thisGameIsRated()),
                     p(
-                      "You must ",
-                      a(
-                        cls := "button",
-                        href := s"${routes.Auth.login}?referrer=${routes.Round.watcher(c.id, "white")}"
-                      )(trans.signIn()),
-                      " to join it."
+                      trans.youMustXToJoinIt(
+                        a(
+                          cls := "button",
+                          href := s"${routes.Auth.login}?referrer=${routes.Round.watcher(c.id, "white")}"
+                        )(trans.signIn())
+                      )
```

Comment:
Can't play lego with translations, it always fails with some languages. Better make a new one, but then the full text would need to be a link/button.
In this case I think just the sign in button without any extra text would be sufficient

Reply:
Done